app/models/mynewcomment_new.rb

```diff
@@ -1,20 +1,20 @@
 # app/models/mynewcomment_new.rb
 
 class MynewcommentNew < ApplicationRecord
-    self.table_name = 'mynewcomments'
-  
-    belongs_to :author, class_name: 'User', foreign_key: 'user_id'
-    belongs_to :post, class_name: 'MynewpostNew'
-  
-    attribute :text, :text
-  
-    after_create :update_comment_counter
-    after_destroy :update_comment_counter
-  
-    def update_comment_counter
-      post.update(comment_counter: post.comments.count)
-    end
+  self.table_name = 'mynewcomments'
+
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post, class_name: 'MynewpostNew'
+
+  attribute :text, :text
+
+  after_create :update_comment_counter
+  after_destroy :update_comment_counter
+
+  def update_comment_counter
+    post.update(comment_counter: post.comments.count)
   end
```

Comment:
- [ ] Kindly include tests for the custom method in your `Comment Model` as well. I have highlighted the method above. So kindly include tests for it :wink: